frontend/src/metabase/pulse/components/PulseEditChannels.jsx

```diff
@@ -153,27 +153,34 @@ export default class PulseEditChannels extends Component {
   renderFields(channel, index, channelSpec) {
     return (
       <div>
-        {channelSpec.fields.map(field => (
-          <div key={field.name} className={field.name}>
-            <span className="h4 text-bold mr1">{field.displayName}</span>
-            {field.type === "select" ? (
-              <Select
-                className="h4 text-bold bg-white"
-                value={channel.details && channel.details[field.name]}
-                options={field.options}
-                optionNameFn={o => o}
-                optionValueFn={o => o}
-                // Address #5799 where `details` object is missing for some reason
-                onChange={o =>
-                  this.onChannelPropertyChange(index, "details", {
-                    ...channel.details,
-                    [field.name]: o,
-                  })
-                }
-              />
-            ) : null}
-          </div>
-        ))}
+        {channelSpec.fields.map(field => {
+          return (
```

Comment:
Why wrap this function in brackets if all you do is return a value?

Reply:
I think this was vestigial from some console.log-ing. I think I'll also add that searchFuzzy here and maybe make it the default? It seems like that provides the nicest out of the box user experience.